src/vm/fieldmarshaler.h

```diff
@@ -89,6 +88,44 @@ enum NStructFieldType
 #define DEFAULT_PACKING_SIZE 32
 
 
+typedef enum
```

Comment:
If we are going to define the `|=` operator, lets make this a proper `enum class`.